docs/usage/security-modes.md

```diff
@@ -0,0 +1,92 @@
+This mode can be useful when the clusters trust each other (and/or belong to the same administrative domain) and there is no need to restrict the connectivity.
+This may happen when both clusters are under control of the same organization, and Liqo is used simply to "cross the boundaries" among distinct clusters (e.g., because clusters are running in different geographical regions), providing a single point of control for many geographically distributed clusters.
+
+In the following picture and table, there will be five pods with different characteristics:
+
+* **P1**: pod in cluster C1, not belonging to the offloaded namespace.
+* **P2**: pod in cluster C1, belonging to the offloaded namespace and used as target by the service S2.
+* **P3**: pod in cluster C1, belonging to the offloaded namespace and not used as target by any service.
+* **P4**: pod in cluster C2, belonging to the offloaded namespace; this pod has been offloaded by cluster C1.
+* **P5**: pod in cluster C2, not belonging to the offloaded namespace; this pod has no relationships with cluster C1 and it is in full control from cluster C2.
+
+In addition, three services (e.g., ClusterIP) are present, with different characteristics:
+
+* **S1**: service under the control of cluster C1, not belonging to the offloaded namespace.
+* **S2**: service under the control of cluster C1, belonging to the offloaded namespace (hence reflected also in C2).
+* **S3**: service under the control of cluster C2, not belonging to the offloaded namespace.
+
+Considering two clusters (C1 and C2) in which the former has started a peering toward the latter, and C1 is using some physical resources on the remote cluster C2, the following picture and table show the connectivity provided by Liqo to each pod when trying to connect to resources (other pods, services, and Internet connectivity) present in either one of the two clusters.
+
+```{figure} /_static/images/usage/security-modes/security-modes-schema.drawio.svg
+---
+align: center
+---
+
+```
+
+```{figure} /_static/images/usage/security-modes/matrix-full-p2p.drawio.svg
+---
+align: center
+---
+
+```
+
+### Intra-cluster traffic segregation
+
+This mode allows the full pod-to-pod connectivity only within each _physical_ cluster, e.g., all pods in the local cluster can talk to each other, and the same holds for all pods in the remote cluster irrespective of the namespace they belong to or whether they are offloaded or not.
+With respect to the intra-cluster traffic, the local cluster can contact only its offloaded pods, while the remote cluster is allowed to contact only the _services_ offloaded on it.
+This mode can be used when the local cluster wants to extend its full pod-to-pod connectivity with pods and services in the offloaded namespace running in the remote cluster, while it wants to limit the remote cluster's connectivity solely to offloaded services, preventing direct interaction with the offloaded pods.
```

Comment:
I cannot understand exactly here.
The sentence says "preventing direct interaction with the offloaded pods". However, the picture shows that P5 (in cluster C2) can contact P4 (pod offloaded by C1). So, either the sentence is imprecise, or the table is wrong.
Comments?

Reply:
The table is right and sentence imprecise.
```suggestion
This mode allows the full pod-to-pod connectivity only within each _physical_ cluster, e.g., all pods in the local cluster can talk to each other, and the same holds for all pods in the remote cluster irrespective of the namespace they belong to or whether they are offloaded or not.
With respect to the intra-cluster traffic, the local cluster can contact only its offloaded pods, while the remote cluster is allowed to contact only the _services_ offloaded on it.
This mode can be used when the local cluster wants to extend its full pod-to-pod connectivity with pods and services in the offloaded namespace running in the remote cluster, while it wants to limit the remote cluster's connectivity solely to offloaded services and related backend pods, preventing interaction with local pods that are not engaged in the multi-cluster topology.
```